pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -233,6 +233,9 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, indent, 0, describeMonopod(f, monopod.Pod)...)
 		}
 
+		bcPipelines, _ := graphview.AllImagePipelinesFromBuildConfig(g, graphview.IntSet{})
```

Comment:
A few thoughts:
1) terminology

   - the subject of the associated issue - making the links between build configs obvious
   - the description that follows mentioned "build chains", but given what @smarterclayton points out, "build chains" implies more than the link between build configs
   - so, if making the link between build configs more obvious is something worth pursuing, back off the term "build chain" in any `oc status` output

2) location

  - I asked for opinions on whether to denote connection in a separate section (perhaps contributing to the redundancy) vs. tweaking the existing umbrella of output
  - so exploring the latter flavor a little, how would changing this output:

```
https://www.example.com (svc/frontend)
  dc/frontend deploys istag/origin-ruby-sample:latest <-
    bc/ruby-sample-build builds https://github.com/openshift/ruby-hello-world.git with pipelineproject/ruby-base-image:latest 
      not built yet
    deployment #1 waiting on image
```  

To this output:

```
https://www.example.com (svc/frontend)
  dc/frontend deploys istag/origin-ruby-sample:latest <-
    bc/ruby-sample-build builds https://github.com/openshift/ruby-hello-world.git with pipelineproject/ruby-base-image:latest (produced by bc/ruby-base) 
      not built yet
    deployment #1 waiting on image
```

be something worth pursuing (and honor the intent of the associated issue)?